docs/operating-eck/troubleshooting/take-eck-dump.asciidoc

```diff
@@ -0,0 +1,125 @@
+:page_id: take-eck-dump
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Take ECK Dump
+
+
+Steps to take ECK dump:
+
+[float]
+== Prepare
+
+To download the ECK dump shell file, run:
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+curl -L -O https://raw.githubusercontent.com/elastic/cloud-on-k8s/master/support/diagnostics/eck-dump.sh
+------------------------------------------------
+
+Change mode to executable.
+
+[source,bash]
+----
+chmod +x eck-dump.sh
+----
+
+[float]
+== Run
+
+Make sure to set the correct options. Check here for link:https://github.com/elastic/cloud-on-k8s/blob/master/support/diagnostics/eck-dump.sh[the latest version].
+
+[source,bash]
+----
+Options:
+  -N, --operator-namespaces     Namespace(s) in which operator(s) are running in (comma-separated list)
+  -n, --resources-namespaces    Namespace(s) in which resources are managed      (comma-separated list)
+  -o, --output-directory        Path to output dump files
+  -z, --create-zip              Create an archive with the dump files (implies --output-directory)
+  -v, --verbose                 Verbose mode
+----
+
+Command
+
+[source,bash]
+----
+./eck-dump.sh -o <path/to/output/dump/files> [-z] [-v] -N <operator namespace> -n <resources namespace>
+----
+
+
+[float]
+== Example
+
+Here is an example.
+
+- Assuming operator & resources are both in `elastic-system` namespace
```

Comment:
I would prefer to use a different namespace than the one used to deploy the operator, to deploy the resources.
And even why not two namespaces:
```
-N "elastic-system" -n "security,monitoring"
```